spec/money/distributed/storage_spec.rb

```diff
@@ -58,7 +58,7 @@
       threads = []
       10.times do
         threads << Thread.new do
-          1000.times do
+          100.times do
```

Comment:
Spec run now in 0.6 seconds vs. 7-8 seconds. 100 should still be good enough for concurrency